### PR TITLE
move composer.json to repo root and support virion 3.0

### PR DESCRIPTION
**Pull request by SOF3**
obsoletes #7

### CODE REVIEW

**Comment by SOF3** [comment]
Hi! After merging this pull request, your users may use Virion 3.0 in their plugins by using `vcs` dependencies in their composer.json, but you could make their lives easier by completing these steps:

1. Tag each release with the `git tag` command or the [GitHub web UI](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
2. Submit your package on Packagist: <https://packagist.org/packages/submit>

Please follow these steps every time you release a new stable version so that user plugins can get their dependencies updated automatically upon the next build.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -26,17 +26,15 @@ jobs:
           php-version: ${{ matrix.php }}
 
       - name: Restore Composer package cache
-        id: composer-cache
+        id: composer-cache-root
```
Comment (dktapps) [inline .github/workflows/main.yml]: I'm not sure what you intended to achieve by changing this

Comment (SOF3) [reply .github/workflows/main.yml]: Oh, I was trying to invalidate the original cache, but turns out it is not necessary anymore and I forgot to revert this.

**Comment by dktapps** [comment]
Is this in line with virion v3 final?

**Comment by SOF3** [comment]
Yes. Please consider tagging a release.

**Pull request merged**